formal-spec/update-protocol.tex

```diff
@@ -117,22 +137,22 @@ \subsection{Activation phase}
 We call a proposal that is in the endorsement period the \emph{candidate
   proposal}.
 
-If the new candidate proposal is a consensus-update, the nodes can start
-\emph{endorsing} it. An endorsement is a special transaction that is included
-into a block and signals the fact that a node has downloaded and installed the
-software that implements the update.
+If the new candidate proposal is a consensus-update, the block producers can
+start \emph{endorsing} it. An endorsement is a special metadata field in the
+block, and can be set only by the block producers. An endorsement signals the
```

Comment:
IMO, after Peter's answer, I would leave endorsements as payload in common transactions and deal with the incentives problem properly at a later stage.